src/pageObject.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
```

Comment:
you can separate parts of the code with an empty line, so the code will be easier to read